### ISSUE TITLE
Issue creating views across databases

### ISSUE THREAD

**Issue by theme1256** [dolthub/dolt#7438]
I can't create a view in one database, that mirrors an existing table in another database, with the same name, without changing context to the other database.

This doesn't work:
```sql
USE db1;
CREATE OR REPLACE VIEW `db2`.`event` AS SELECT * FROM `db1`.`event`;
```

This works:
```sql
USE db2;
CREATE OR REPLACE VIEW `db2`.`event` AS SELECT * FROM `db1`.`event`;
```

**Comment by timsehn** [comment]
This is a bit of a gap for Dolt. @zachmu spent a bunch of time making a lot of multi-db syntax work but there are still gaps. We'll patch this one up.

**Comment by fulghum** [comment]
Thanks for reporting this one @theme1256! 🙏  Another good find!

I debugged through your repro and found where we are grabbing the wrong database to create the view. I've got a fix in my workspace and am starting on the tests now. I'll get a PR opened asap and we'll get this one fixed up for you!

**Comment by bpf120** [comment]
@theme1256 , Thanks again for all the bugs! Hope we can learn about your use case soon. :) Feel free to  [email me](mailto:brianf@dolthub.com) or swing by our [Discord](https://discord.gg/gqr7K4VNKe) if you want to share.

**Comment by fulghum** [comment]
Thanks for helping us find this one @theme1256! 🙏  We just released the fix in [Dolt 1.32.5](https://github.com/dolthub/dolt/releases/tag/v1.32.5).

Let us know if you run into any other issues and we'll be happy to help.
